docs_source_files/content/webdriver/keyboard.en.md

```diff
@@ -3,3 +3,42 @@ title: "Keyboard"
 weight: 10
 ---
 
+Represents a KeyBoard event. KeyBoard actions are performed by using low-level
+interface which allows us to provide virtualized device input to the web browser.
+
+# sendKeys
```

Comment:
Something I missed, and it is already merged, this should not be an `# sendKeys` but instead a `## sendKeys`.

Could you please send another PR with that change @Harsha509? Thanks!

Reply:
Sure. i'll send another PR.

Reply:
@diemol, Updated in PR https://github.com/SeleniumHQ/docs/pull/351

I'll be cautious next time.

Thanks!